### PR TITLE
python language updates

### PR DESCRIPTION
**Pull request by marksmayo**
upgraded a bunch of format strings to f-strings
formatted with black afterwards to meet previous style

all tests pass (Windows 11)

### CODE REVIEW

**Comment by keikoro** [comment]
@marksmayo Thanks for your PR!

The Flake8 test seems to be failing due to overly complicated / redundant string concatenation, [more info here](https://github.com/flake8-implicit-str-concat/flake8-implicit-str-concat). If you could fix those, that'd be great.

**Comment by keikoro** [comment]
I don't know why the black check didn't end up failing for your PR because it did for the most recent previous ones, but either way: it might make sense to update to the latest commit on `master` (which fixes that failing check), just in case...

**Review by ohld — APPROVED** [review]

File: __temp__.mp3
Comment (mgaitan) [inline __temp__.mp3:1]: these top-level files shouldn't be commited, right?

File: moviepy/audio/io/readers.py [hunk 1]
```diff
@@ -179,9 +179,9 @@ def get_frame(self, tt):
             # Check that the requested time is in the valid range
             if not in_time.any():
                 raise IOError(
-                    "Error in file %s, " % (self.filename)
-                    + "Accessing time t=%.02f-%.02f seconds, " % (tt[0], tt[-1])
-                    + "with clip duration=%f seconds, " % self.duration
+                    f"Error in file {self.filename}, "
+                    + f"Accessing time t={tt[0]:.02f}-{tt[-1]:.02f} seconds, "
+                    + f"with clip duration={self.duration:f} seconds, "
```
Comment (mgaitan) [inline moviepy/audio/io/readers.py:184]: you could avoid the `+` and as the concatenation will be implicit 
https://docs.python.org/3/reference/lexical_analysis.html#string-literal-concatenation

File: moviepy/audio/io/readers.py [hunk 2]
```diff
@@ -202,10 +202,10 @@ def get_frame(self, tt):
 
             except IndexError as error:
                 warnings.warn(
-                    "Error in file %s, " % (self.filename)
-                    + "At time t=%.02f-%.02f seconds, " % (tt[0], tt[-1])
+                    f"Error in file {self.filename}, "
+                    + f"At time t={tt[0]:.02f}-{tt[-1]:.02f} seconds, "
                     + "indices wanted: %d-%d, " % (indices.min(), indices.max())
-                    + "but len(buffer)=%d\n" % (len(self.buffer))
+                    + f"but len(buffer)={len(self.buffer)}\n"
```
Comment (mgaitan) [inline moviepy/audio/io/readers.py:208]: remove the `+` operators

**Comment by keikoro** [comment]
I'm confused about how someone who isn't a maintainer of the project (as far as I can see) ended up as a reviewer for this PR – and without GitHub providing any meta info on that, no less (normally, it should say something like "X requested a review by Y" or similar, no?) – or why they were allowed to approve the changes. Very curious.

**Comment by OsaAjani** [comment]
Thank you for your contributions and for reporting issues in this repository. With the release of v2, which introduces significant changes to the codebase and API, we’ve reviewed the backlog of open PRs and issues. Due to the length of the backlog and the likelihood that many of these are either fixed or no longer applicable, we’ve made the decision to close all previous PRs and issues.

If you believe that any of these are still relevant to the current version or if you'd like to reopen a related discussion, please feel free to create a new issue or pull request, referencing the old one.

Thank you for your understanding and continued support!

**Pull request closed without merging**